src/test/scala/uk/gov/hmrc/api/Specs/tags/Ignore.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.api.Specs.tags
+
+import org.scalatest.Tag
+
+object Ignore extends Tag("Ignore")
```

Comment:
And do we need this ignore tag? The only place I can see it being used is in `ExternalStubsScenarioSpec`, where it has been commented out. See the comment there for a suggestion on how to ignore stuff in scalatest.